pliers/extractors/text.py

```diff
@@ -313,3 +335,38 @@ def _extract(self, stim):
         features = ['sentiment_' + k for k in scores.keys()]
         return ExtractorResult([list(scores.values())], stim, self,
                                features=features)
+
+class SpaCyExtractor(TextExtractor):
+    
+    def __init__(self, model='en_core_web_sm', features=None):
+        #verify_dependencies(['spacy'])
+        try: 
+            nlp = spacy.load('en_core_web_sm')
```

Comment:
I imagine this was probably just a first step, but don't forget to replace `'en_core_web_sm'` with `self.model`, as we want to load whatever model the user wants to use.